test/karma-setup.js

```diff
@@ -85,14 +85,25 @@ QUnit.module('Highcharts', {
 
     afterEach: function () {
 
-        // Destroy all charts
-        Highcharts.charts.forEach(function (chart) {
-            if (chart && chart.destroy && chart.renderer) {
-                chart.destroy();
+        var currentChart = null,
+            charts = Highcharts.charts,
+            templateCharts = [];
+
+        // Destroy all charts, except template charts
+        for (var i = 0, ie = charts.length; i < ie; ++i) {
+            currentChart = charts[i];
+            if (!currentChart) {
+                continue;
             }
-        });
-        Highcharts.charts.length = 0;
+            if (currentChart.template) {
+                templateCharts.push(currentChart);
+            } else if (currentChart.destroy && currentChart.renderer) {
+                currentChart.destroy();
+            }
+        }
 
+        Highcharts.charts.length = 0;
+        Array.prototype.push.call(Highcharts.charts, templateCharts);
```

Comment:
Shouldn't be `apply` instead of `call`? 

```
var a = [], b = [1, 2, 3];
Array.prototype.push.call(a, b);
```
> a = [[1, 2, 3]]


```
var a = [], b = [1, 2, 3];
Array.prototype.push.apply(a, b);
```
> a = [1, 2, 3]

Reply:
Ooops. Thanks! I will fix this.